### DISCUSSION TITLE
We're still processing your application. Once this is complete you will have access to the student developer pack offers for 5 months now

### DISCUSSION THREAD

**Discussion by PranavJaju** [community/community#121804] [GitHub Education]
### Hi everyone,

We're still processing your application. Once this is complete you will have access to the student developer pack offers 

I am getting this notification for nearly 5 months still I am not been able to get access of the pack 

![image](https://github.com/community/community/assets/100665512/213ce32a-6826-4e08-bef2-fa0e3c325cf2)

I had got approved on 20th Jan, 2024, so far I haven't received any form of communication from the GitHub Team
I would appreciate if some one could guide me about how to claim the pack benefits 
thank you

**Comment by gurjeetsinghvirdee** [comment]
Hey @PranavJaju 

please go through this, maybe you will find your answer here #111352
